pkg/archive/archive.go

```diff
@@ -405,6 +405,8 @@ func Tar(path string, compression Compression) (io.ReadCloser, error) {
 // paths are included in `options.IncludeFiles` (if non-nil) or not in `options.ExcludePatterns`.
 func TarWithOptions(srcPath string, options *TarOptions) (io.ReadCloser, error) {
 
+	srcPath = fixVolumePathPrefix(srcPath)
```

Comment:
Sort of... you can't use <code>'\\\\?\'</code> and any relative path stuff (no "." or "..").  That's why we also needed to use filepath.Join below, that way those artifacts get pruned out and an absolute path is returned.